configurations/admin.template.ini

```diff
@@ -165,12 +149,20 @@ moduls.liveStreamRecordShouldCopyEntitelment.group = GROUP_ENABLE_DISABLE_FEATUR
 
 moduls.kalturaLiveStreamTranscode.enabled = true
 moduls.kalturaLiveStreamTranscode.permissionType = 2
-moduls.kalturaLiveStreamTranscode.label = Kaltura Live Streams Plus
+moduls.kalturaLiveStreamTranscode.label = Kaltura Live SD Flavors
 moduls.kalturaLiveStreamTranscode.permissionName = FEATURE_KALTURA_LIVE_STREAM_TRANSCODE
 moduls.kalturaLiveStreamTranscode.basePermissionType = 2
 moduls.kalturaLiveStreamTranscode.basePermissionName = FEATURE_KALTURA_LIVE_STREAM
 moduls.kalturaLiveStreamTranscode.group = GROUP_ENABLE_DISABLE_FEATURES
 
+moduls.liveHdFlavors.enabled = true
+moduls.liveHdFlavors.permissionType = 2
+moduls.liveHdFlavors.label = Kaltura Live HD Flavors
+moduls.liveHdFlavors.permissionName = FEATURE_KALTURA_LIVE_STREAM_HD_FLAVORS
```

Comment:
Rename to FEATURE_KALTURA_LIVE_STREAM_HD_TRANSCODE to be consistent with FEATURE_KALTURA_LIVE_STREAM_TRANSCODE